src/stan-users-guide/odes.Rmd

```diff
@@ -17,364 +16,253 @@ for stiff systems.
   [@CohenHindmarsh:1996; @SerbanHindmarsh:2005]
 
 For a discussion of stiff ODE systems, see the [stiff ODE
-section](#stiff-ode.section).  In a nutshell, the stiff solvers are
-slower, but more robust; how much so depends on the system and the
-region of parameter space.  The function signatures for Stan's ODE
+section](#stiff-ode.section). The function signatures for Stan's ODE
```

Comment:
> This is due to the attempt of Adams to do a higher order approximation of the function in order to make large steps and in turn avoid excessive calls to the ODE RHS (whereas RK45 just calls brutally the ODE RHS very often).

I think we should work on a separate workflow ODE document because we have no real guidance on picking between the solvers. I'd like to just leave this for the future.

Reply:
I find the text as is confusing as there is no guidance at all to pick between the two non-stiff ones. I would prefer to tell people: Start with RK45 in any case; if stiffness is a concern go with bdf and whenever the problem is non-stiff and the ODE RHS is expensive to evaluate, then try Adams (after having tried RK45).

This would reflect my intuition and I think is a fair summary.  If you think we are not ready for this... ok.